WickedEngine/wiScene_BindLua.cpp

```diff
@@ -701,6 +702,35 @@ int Scene_BindLua::Merge(lua_State* L)
 	return 0;
 }
 
+int Scene_BindLua::Entity_FindAllEntities(lua_State* L)
+{
+
+	int argc = wi::lua::SGetArgCount(L);
+	if (argc > 0)
+	{
+		wi::lua::SError(L, "Scene::Entity_FindAllEntities() doesnt take any argument");
```

Comment:
It's ok if you provide parameters to a function that doesn't take any, it shouldn't be an error. All lua bindings follow this convention currently.

Reply:
it's okay then i will remove that part , but is there still a way to provide some sort of a warning ? for example wi::lua::SWarning("Using too much arguments") for example ?